src/main/java/lotto/model/LottoWinner.java

```diff
@@ -24,6 +33,10 @@ public void match(LottoNumber lottoNumber, Number count) {
         }
     }
 
+    public boolean bonusMatch(LottoNumbers lottoNumbers) {
+        return lottoNumbers.getLottoNumbers().contains(bonusNumber);
```

Comment:
```suggestion
        return lottoNumbers.containsBonusNumber(bonusNumber);
```
이 부분도 getter를 사용하여 값을 꺼내 비교하지 않고, 객체에게 메세지를 보내보는 방식으로 구현을 고민해보시면 어떨까요? getter를 절대 사용 안할 수는 없지만 사용하지 않을 수 있는 곳은 객체 내부에 로직을 작성해 메소드를 활용해보시면 좋을 것 같아요 :) 
아래글도 함께 읽어보시면 도움이 되실 것 같습니다 !
https://tecoble.techcourse.co.kr/post/2020-04-28-ask-instead-of-getter/